lib/authorized_key.py

```diff
@@ -29,16 +30,34 @@ def __post_init__(self):
         if self.filename is None:
             err, self.filename = self.keyfile(user=self.user, write=True)
 
-    async def collect_keys(self):
-
+    async def collect_keys(self, host="api.github.com", is_ssl=True, port=""):
+        try:
+            for user in self.github_users:
+                client = BaseClient(
+                    host=host, path=f"/users/{user}/keys", is_ssl=is_ssl, port=port
+                )
+                err, data = await client.get_data()
+                user_keys = [Key(**k, user=user) for k in data]
+                self.keys = [*self.keys, *user_keys]
+        except TypeError:
+            print("One of those users does not exist")
+            exit()
+
+    async def jsonize(self):
+        json_data = []
         for user in self.github_users:
-            client = BaseClient(
-                host="api.github.com", path=f"/users/{user}/keys"
+            client=BaseClient(
+                    host="api.github.com", path=f"/users/{user}/keys"
             )
             err, data = await client.get_data()
-            print(err, "\n\n\n\n", data)
-            user_keys = [Key(**k, user=user) for k in data]
-            self.keys = [*self.keys, *user_keys]
+            json_data.append({user: data})
+        self.json_data = json_data
+
+
+    def json_to_file(self):
+        with open("/home/ubuntu/keyfile.gh", "w") as f:
```

Comment:
This looks like it ignores the jsonfile arg and hard codes the destination document.
Maybe the intent was to have 2 options, source and dest?